raiden/raiden_service.py

```diff
@@ -983,7 +983,7 @@ def message_mediatedtransfer(self, message):
 
         channel = graph.partneraddress_channel[message.sender]
 
-        if not channel.isopen:
+        if not channel.can_transfer:
```

Comment:
Same thing with the direct transfer, a channel must accept a message if it can't transfer